docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -379,182 +1209,733 @@ If you run `kubectl get pods -n sample-domain1-ns --watch`, then you should see
-   > GET /weblogic/ready HTTP/1.1
-   > User-Agent: curl/7.29.0
-   > Accept: */*
-   > host: sample-domain1.org
-   >
-   < HTTP/1.1 200 OK
-   < Content-Length: 0
-   < Date: Mon, 09 Mar 2020 20:40:37 GMT
-   < Vary: Accept-Encoding
-   <
-   * Connection #0 to host myhost.my.dns.domain.name left intact
+   kubectl -n sample-domain1-ns delete configmap sample-domain1-wdt-config-map --ignore-not-found
+   kubectl -n sample-domain1-ns create configmap sample-domain1-wdt-config-map --from-file=/tmp/mii-sample/model-configmaps/datasource
+   kubectl -n sample-domain1-ns label  configmap sample-domain1-wdt-config-map weblogic.domainUID=sample-domain1
    ```
 
-   **Note**: If you're running on a remote Kubernetes cluster, then substitute `$(hostname).$(dnsdomainname)` with an external address suitable for contacting the cluster.
+   - If you've created your own datasource file, then substitute the file name in the `--from-file=` parameter. 
+     - Note that the `-from-file=` parameter can reference a single file, in which case it puts the designated file in the configmap, or it can reference a directory, in which case it populates the configmap with all of the files in the designated directory.
 
-#### Optionally access the WebLogic Server Administration Console
+   - About deleting and recreating the configmap:
+     - We delete a configmap before creating it, otherwise the create command will fail if the configmap already exists.
+     - This allows us to change the configmap when using the `kubectl create configmap` verb.
 
-{{% notice warning %}} This sample externally exposes the WebLogic Server Administration Console using a plain text HTTP port. This is _not_ secure and should not be done in production deployments.
-{{% /notice %}}
+   - We name and label configmap using their associated domain UID for two reasons:
+     - To make it obvious which configmap belong to which domains.
+     - To make it easier to cleanup a domain. Typical cleanup scripts use the `weblogic.domainUID` label as a convenience for finding all resources associated with a domain.
 
-You can add an Ingress rule to access the WebLogic Server Administration Console from your local browser.
+   {{% /expand%}}
 
-1. Find out the service name of the Administration Server and service port number.
+1. _Update your domain resource file to refer to the configmap and its secret, and apply it._
 
-   The service name follows the pattern `domainuid-adminservername`, all lower case, with a hyphen `-` substituted for each underscore `_`.
-   The port number for administration traffic is configured in your WebLogic configuration (your model files), where the default is `7001`.
+      Do one the following:
 
-   If your domain resource is deployed, then you can also find the information by getting the port number for the Administration Server pod:
+      {{%expand "Click here to expand." %}}
+  - Option 1: Update your current domain resource file from the Initial use case.
```

Comment:
i could have used a reminder here of what i called that file :)

Reply:
done.